### DISCUSSION TITLE
Button/Link to quickly set datetime field to current date time

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#4195] [Q&A] [answered]
for date field I can still click on the current date on the calendar widget

but for datetime field, I cannot quickly fill the current time. Only the date portion

**Comment by crynobone** [comment, marked as answer]
Date uses `<input type="date">` and DateTime uses `<input type="datetime-local">`. Both datepicker are handled directly by each browser and will have different UI
